microsoft-edge/extensions-chromium/whats-new/released-features.md

```diff
@@ -38,6 +58,19 @@ Microsoft Edge Add-ons now support extensions with Manifest V3. Check out [the b
 
 *Released April 2021*
 
+<!-- ====================================================================== -->
+## Request to get your extension featured
+
+Developers can request the Collection on which the extension should be featured on the Microsoft Edge Add-ons store. You can submit your request [here](https://aka.ms/MSEACollections).
+
+*Released April 2021*
+
+<!-- ====================================================================== -->
+## Extension store listing localization
+
+Developers can choose a subset of languages instead of manually entering each language in the listing details on Partner Center & the ability to use the same marketing assets across all marketplaces. Check out [the blog](https://techcommunity.microsoft.com/t5/articles/partner-center-improvements-related-to-microsoft-edge-extensions/m-p/2118981) for more details.
```

Comment:
```suggestion
Developers can choose a subset of languages instead of manually entering each language in the listing details on Microsoft Partner Center and the ability to use the same marketing assets across all marketplaces. Check out [the blog](https://techcommunity.microsoft.com/t5/articles/partner-center-improvements-related-to-microsoft-edge-extensions/m-p/2118981) for more details.
```